apps/dapp-console-api/src/testhelpers/auth.ts

```diff
@@ -1,21 +1,16 @@
-import type { PrivyClient } from '@privy-io/server-auth'
 import type { AnyRouter } from '@trpc/server'
 import { createCallerFactory } from '@trpc/server'
+import bcrypt from 'bcrypt'
 import type { Request, Response } from 'express'
 import type { getIronSession } from 'iron-session'
-import { type Mock, vi } from 'vitest'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { Mock } from 'vitest'
 
 import type { SessionData } from '@/constants'
-import { PRIVY_TOKEN_COOKIE_KEY } from '@/constants'
+import { envVars, PRIVY_TOKEN_COOKIE_KEY } from '@/constants'
 import type { Session } from '@/constants/session'
-import { getEntity, insertEntity } from '@/models'
 
-vi.mock('../models', async () => ({
```

Comment:
this was causing a bunch of issues with trying to use the helpers in this file while also having a different set of mocks, so removing entirely.